README.md

```diff
@@ -40,15 +43,42 @@ It’s a Rails app that:
     $ bundle install --without production
     ```
 
-7. Set up your database. The simple way to do this is:
+7. Set up your database
 
+    Make a local copy of the database configuration template file:
+    
     ```sh
-    $ bundle exec rake db:setup
+    cp config/database.yml.example config/database.yml
+    ```
+
+    First create users, databases and configure extensions:
+
+    ```sh
+    sudo -u postgres psql -c "CREATE USER wmdb_dev WITH PASSWORD 'wmdb_dev';"
+    sudo -u postgres createdb -O wmdb_dev wmdb_dev -E utf-8
+    sudo -u postgres psql -c 'CREATE EXTENSION IF NOT EXISTS "uuid-ossp";' wmdb_dev
+    sudo -u postgres psql -c 'CREATE EXTENSION IF NOT EXISTS "plpgsql";' wmdb_dev
```

Comment:
If your postgres user is not a superuser then it won't have rights to create extensions through schema load and you need to setup them via `postgres` superuser in this way.